docs/docs/reference/core-plugins/elasticsearch-plugin/index.md

```diff
@@ -34,27 +51,57 @@ Make sure to remove the `DefaultSearchPlugin` if it is still in the VendureConfi
 
 Then add the `ElasticsearchPlugin`, calling the `.init()` method with <a href='/reference/core-plugins/elasticsearch-plugin/elasticsearch-options#elasticsearchoptions'>ElasticsearchOptions</a>:
 
-*Example*
+_Example_
 
 ```ts
 import { ElasticsearchPlugin } from '@vendure/elasticsearch-plugin';
 
 const config: VendureConfig = {
-  // Add an instance of the plugin to the plugins array
-  plugins: [
-    ElasticsearchPlugin.init({
-      host: 'http://localhost',
-      port: 9200,
-    }),
-  ],
+    // Add an instance of the plugin to the plugins array
```

Comment:
These docs are generated, so you should not edit them directly. Instead, you need to edit the [doc block of the plugin source](https://github.com/vendure-ecommerce/vendure/blob/master/packages/elasticsearch-plugin/src/plugin.ts#L83) and add the changes there. Next time the docs are generated, these changes will be added to the md files.